components/reports/PrecipitationFrequency.vue

```diff
@@ -0,0 +1,242 @@
+      </div>
+    </div>
+    <h4 class="title is-4 mt-6">
+      Projected precipitation frequency, {{ radioModel }}, {{ radioEra }}
+    </h4>
+    <table class="table">
+      <thead>
+        <tr>
+          <th class="no-border">Duration</th>
+          <th class="no-border" colspan="9">
+            Annual exceedance probability (1/years)
+          </th>
+        </tr>
+        <tr>
+          <th></th>
+          <th v-for="interval in [2, 5, 10, 25, 50, 100, 200, 500, 1000]">
+            1 / {{ interval }}
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr
+          v-for="duration in [
+            '60m',
+            '2h',
+            '3h',
+            '6h',
+            '12h',
+            '24h',
+            '2d',
+            '3d',
+            '4d',
+            '7d',
+            '10d',
+            '20d',
+            '30d',
+            '45d',
+            '60d',
+          ]"
+        >
+          <td>{{ duration }}</td>
+          <td v-for="interval in [2, 5, 10, 25, 50, 100, 200, 500, 1000]">
+            {{
+              results.proj_precip[
+                `pr_${interval}_${duration}_${radioModel}_${radioEra}_mean`
+              ]
+            }}<UnitWidget unitType="mm_in" /><br />
+            <span class="small-text">
+              {{
+                results.proj_precip[
+                  `pr_${interval}_${duration}_${radioModel}_${radioEra}_min`
+                ]
+              }}&mdash;{{
+                results.proj_precip[
+                  `pr_${interval}_${duration}_${radioModel}_${radioEra}_max`
+                ]
+              }}
+            </span>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <div class="block data-outro content is-size-5 no-print">
+      <h4 class="title is-5 no-print">
+        Data access &amp; additional information
+      </h4>
+
+      <ul>
+        <li>
+          <DownloadCsvButton
+            text="Download this data as CSV"
+            endpoint="proj_precip/point"
+          />
+        </li>
+        <li>
+          Read a
+          <a
+            href="https://uaf-snap.org/wp-content/uploads/2021/05/dot-precip_FINAL-REPORT_20210526.pdf"
+            >detailed report and overview of the data preparation and research
+            methodology</a
+          >
+          used to prepare this dataset
+        </li>
+        <li>
+          Academic reference: Bieniek P, Walsh J, Fresco N, Tauxe C, Redilla K.
+          Anticipated changes in Alaska extreme precipitation. Journal of
+          Applied Meteorology and Climatology. 2022; 61(2):97-108.
+          <a href="https://doi.org/10.1175/JAMC-D-21-0106.1"
+            >https://doi.org/10.1175/JAMC-D-21-0106.1</a
+          >
+        </li>
+        <li>
+          Source dataset and metadata:
+          <a
+            href="https://catalog.snap.uaf.edu/geonetwork/srv/eng/catalog.search#/metadata/304b6d89-961e-417d-b6ba-4139c7fe5ff6"
+            target="_blank"
```

Comment:
```suggestion

```

Per our previous conversations, do we want this to not open a new tab now that we modified the code to allow for the back button to initiate a re-request of the backend API data?